app/Domain/Auth/Controllers/Callback.php

```diff
@@ -21,7 +22,13 @@ public function __invoke(Request $request): RedirectResponse
             $user = $auth0->getUser();
 
             if ($user !== null) {
+                // httpOnly = false
+                Cookie::queue('auth.token.idToken', $auth0->getIdToken(), 3600 * 60 * 24, null, null, null, false);
```

Comment:
I'm not sure this is the correct location to create the cookie. Because the callback is only reached when the user needs to log in. If we start working with a remember me then this callback endpoint will not be reached.

I think some kind of middleware is a better place.

Reply:
moved it to the middleware (and learned about middleware in Laraval at the same time)